scala/ppml/src/main/scala/com/intel/analytics/bigdl/ppml/crypto/BigDLEncrypt.scala

```diff
@@ -35,13 +35,13 @@ import scala.util.Random
 /**
  * BigDL general crypto for encrypt and decrypt data.
  */
-class BigDLEncrypt extends Crypto {
-  protected var cipher: Cipher = null
+class BigDLEncrypt(enableNativeAESCBC: Boolean = false) extends Crypto {
+  var cipher: Cipher = null
   protected var mac: Mac = null
-  protected var ivParameterSpec: IvParameterSpec = null
```

Comment:
why delete protected?

Reply:
‘cause it needs to be set by an external class [here](https://github.com/intel-analytics/BigDL/blob/ee51901727a30279209a45467b8e8ee4b59c21ab/scala/ppml/src/main/scala/com/intel/analytics/bigdl/ppml/crypto/CryptoCodec.scala#L144)

Reply:
going to append a set method